SofaKernel/framework/sofa/defaulttype/DataTypeInfo.h

```diff
@@ -227,14 +227,19 @@ class AbstractTypeInfo
     virtual std::string name() const = 0;
 
     /// True iff the TypeInfo for this type contains valid information.
+    /// A Type is considered "Valid" if there's at least one specialization of the ValueType
```

Comment:
I don't remember in doxygen, is it possible to put tag likes @sa ValuteType to link to the object doc?